infrastructure/code/issue_enhancer.go

```diff
@@ -187,15 +189,18 @@ func (b *IssueEnhancer) autofixFunc(ctx context.Context, issue snyk.Issue,
 		defer pollingTicker.Stop()
 		timeoutTimer := time.NewTimer(2 * time.Minute)
 		defer timeoutTimer.Stop()
+		tries := 1.0
 		for {
 			select {
 			case <-timeoutTimer.C:
 				log.Error().Str("method", "GetAutofixSuggestions").Str("requestId", b.requestId).Msg("timeout requesting autofix")
 				b.notifier.SendShowMessage(sglsp.MTError, "Something went wrong. Please try again. Request ID: "+b.requestId)
 				return nil
 			case <-pollingTicker.C:
+				p.ReportWithMessage(int(math.Max(tries, 99)), "Polling for fix...")
 				fix, complete := pollFunc()
 				if !complete {
+					tries++
```

Comment:
Is this logic to set a ceiling to the max number or retries?

Reply:
Nope, it's just to not report more than a 100% (timeout is currently set to 120, and didn't want to implement a complicated logic with rounding to integers afterwards).